spec/classes/archive/commmon_spec.rb

```diff
@@ -6,6 +6,24 @@
   on_supported_os.each do |os, facts|
     context "on #{os}" do
       let(:facts) { facts }
+      let(:manifest) do
```

Comment:
Why is this needed?

Reply:
to test
https://github.com/lsst-it/lsst-itconf/blob/IT-4339_bts/site/profile/manifests/archive/common.pp#L35-L57

Reply:
A manifest block adds resources into the test catalog but it doesn't create a test.  This appears unnecessary and I believe it can be removed.